packages/main/src/MultiComboBox.js

```diff
@@ -585,6 +585,29 @@ class MultiComboBox extends UI5Element {
 		this[`_handle${event.key}`] && this[`_handle${event.key}`](event);
 	}
 
+	_handleShow(event) {
```

Comment:
Let's discuss with the team one scenario which is not valid UI5 so we can not use it as a reference: When we type a value that does not match a suggestion, on Alt + Down/F4 should we open an empty picker and the focus remains in the input field as it currently happens